packages/moonstream-components/src/core/providers/Web3Provider/index.tsx

```diff
@@ -97,13 +102,14 @@ export const chains: { [key in supportedChains]: ChainInterface } = {
 };
 
 const isKnownChain = (_chainId: number) => {
+  let isKnownChain = false;
```

Comment:
Is `return Object.keys(chains).map((key) => { return chains[key].chainId; }).includes(_chainId);` a simpler way to write this?

Reply:
Your proposal would be just creating another array instance and making one additional iteration.

Reply:
Yes, but it's a very small list. I think the `Object.keys(chains).find((key) => { return chains[key].chainId == _chainId })` solution works here as well.